scripts/iota_cache.py

```diff
@@ -0,0 +1,67 @@
+import time
+import sys
+from iota import Iota, Address, ProposedTransaction, Tag, Transaction, TryteString, TransactionTrytes, ProposedBundle
+from six import binary_type, moves as compat, text_type
+
+
+class IotaCache:
+    def __init__(self, uri=None, seed=None):
+        if not uri:
+            self.uri = "http://localhost:14700"
+        if not seed:
+            self.seed = 'EBZYNR9YVFIOAZUPQOLRZXPPPIKRCJ9EJKVCXMYVLMNOCCOPYPJKCWUZNLJZZZZWTMVQUXZFYLVLZXJ9Q'
+        self.api = Iota(self.uri, self.seed)
+
+    def cacheTxnInTangle(self, IPFSAddr, tag):
```

Comment:
May be use cache_txn_in_tangle to name the this method is better?
Also use ipfs_addr to name the parameter is better?